### PR TITLE
controller: new release

### PR DESCRIPTION
**Pull request by gsanchietti**
New controller

Before merge, add a link to NS8 documentation.

Card: https://trello.com/c/CZeUBCkQ/390-controller-publish-ns8-package

### CODE REVIEW

File: controller.rst [hunk 1]
```diff
@@ -4,26 +4,196 @@
 Controller
 ==========
 
-The NethSecurity controller is an application that can run on `NethServer 8 <https://ns.nethserver.org>`_.
-The controller enables to manage multiple NethSecurity installations remotely.
+The NethSecurity controller is an application that can run on `NethServer 8 (NS8) <https://ns.nethserver.org>`_.
+The controller enables to manage multiple NethSecurity installations remotely, called units.
```
Comment (filippocarletti) [inline controller.rst:8]:
```suggestion
The controller allows remote control of multiple NethSecurity installations, called units.
```

File: controller.rst [hunk 2]
```diff
@@ -4,26 +4,196 @@
...
 
-Firewalls can connect to the controller. Upon successful registration, the server performs the following actions:
+The firewall can operate independently without the need for the controller.
+The controller is an optional component that provides additional management and monitoring capabilities for the firewall.
 
-- generates a VPN configuration and transmits it to the firewall
+The controller works by creating a secure connection between the central server and the units.
+Each firewall registers with the server using a client called ns-plug.
+Once registered, the server generates a VPN configuration that is sent back to the firewall
+This allows for secure communication between the controller and the unit.
```
Comment (filippocarletti) [inline controller.rst:16]:
```suggestion
The VPN enables secure communication between the controller and the unit.
```

File: controller.rst [hunk 3]
```diff
@@ -4,26 +4,196 @@
...
+- `Let's Encrypt certificate`: Enable or disable Let's Encrypt certificate for the controller web interface.
+- `VPN details`: The OpenVPN network and netmask. When choosing the network, make sure it does not overlap with the existing networks inside all
+  the units that will be connected to the controller.
+- `VPN common name`: The OpenVPN certificate name which is also the name of the controller.
+- `Administrator user and password`: The default controller admin user and password. You should change the password after the first login both
+  for the controller and the Grafana interface.
+- `Log retention`: The log retention period in days, default is 180 days.
+- `Metrics retention`: The metrics retention period in days, default 15 days.
+
+After the configuration is done, the controller is ready to be used and can be accessed using a web browser at the configured hostname, like ``https://mycontroller.nethsecurity.org``.
```
Comment (filippocarletti) [inline controller.rst:47]:
```suggestion
After completing the configuration, the controller is ready to be used and can be accessed using a web browser at the configured hostname, like ``https://mycontroller.nethsecurity.org``.
```

File: controller.rst [hunk 4]
```diff
@@ -4,26 +4,196 @@
...
+Users
+=====
+
+The controller has two types of users:
+
+- **Administrator**: The administrator user is the only one that can create and manage users inside the controller.
+- **User**: The user can manage the units and the firewall configurations.
+
+The administrator user is created during the controller configuration. 
+The administrator user is the only one who can create and manage users within the controller.
```
Comment (filippocarletti) [inline controller.rst:58]: Repeated from above: remove?

**Review by filippocarletti — CHANGES_REQUESTED** [review]

File: controller.rst (hunk 4 above)
Comment (gsanchietti) [inline controller.rst:58]:
```suggestion
```

Comment (gsanchietti) [reply controller.rst]: Removed

**Review by filippocarletti — APPROVED** [review]

**Pull request merged**